balancer/rls/internal/keys/builder.go

```diff
@@ -135,7 +139,7 @@ func (bm BuilderMap) RLSKey(md metadata.MD, host, path string) KeyMap {
 		kvMap[b.hostKey] = host
 	}
 	if b.serviceKey != "" {
-		kvMap[b.serviceKey] = service
+		kvMap[b.serviceKey] = service[1 : len(service)-1]
```

Comment:
Hmm ... Actually was your comment to replace this line with `strings.Trim(service, "/")`. I guess I can do that.

Reply:
Done :)